lib/imgix-loader.ts

```diff
@@ -12,4 +12,4 @@ export const imgixLoader = ({ src, width, quality }: ImageLoaderProps) => {
ScDvSaFLwxU7iZwOOXveS2ucimkrjkzHpcccGZuPLH8zmxGBl2KqqGj0fE0GDqKnRNwXE1KrusxJDrJuv4p6vuhKqbmDVrNlS7FapmwlGiiKgeJgzwsSlVcVCAFHw5WYCvr9WQSWfQ7sZZHd37SvOXB5BinqYyWVSUxHDcOTPRv6oCD/7nv6Gish90w8QFN/4XrGwaj82ZgdlvlUFTVAaEMsFUhDqhRoXt+rj2FAPZdAbvr/8Ecx77G9/+qV9chZW7yzF9qI1WO46BJR7nyIonRW9qXma8um8hj6iW/uyqNixe047bzh3x5QRkZWMKo8stLs3G43FkhckFIspxfeuH9yLZ0YZFD89GeWUVbn85jbY0GAxiCdnPcTUaTNXDdyeYaGzajVt+9WeWpqKouO/Wi5A0h6At5eLkWhfxRKRbHNKzflOcdQidkO7gKLh1YQscUOCqfPkAIXX19hYXY0r3oqqqioEIt0w2iymX/xTVFSV4/Nc/QiQaZfmRU/Ovz2U5G6xDplXqhmhwXA9dbbux/NVFsDJdMKMJRBJlsDIpVNYMxpBxXwcUQ3aMUKOET+08PufDKCsgv8unU/aBjlHeS2YhpHqlUi/VRRRN59z9F96o9+ZSOI7DQFA5NdxIUBMvmQ07l8VfH/kRzPIBHAyGgIRe2r88m+OgTYOCqcM15LI5vPr8H7iqZ2WSSJRWsvAi8VKMO+MyuEKHcH04kAAI32cXNw8Mp1HoOTKY4qa4oNBHz6SwUuNJQ7m0LyFDegPI8zx4ro0J374DL/32JgwcWItoNMpDW1I+z17Z3CDjGEpKXvu8DU1XMbFWwavPPsRj0x17eJbHSitQPWgkRk87T3asOD4calhw6V6COxSJCSRsKg0TGwggylf5AWOFUCVLqMJIURBVtqgY9kV3ew/E6SbG7MnqcDu3o3//oxGJRPa5bEeXzEkRMGHGef4KBw1NPjre+b10rX0fZRVV2PHJOpxx9R2IxsvgOAJzzjIP5Gfkx1z+ZDuEEkTa1N1CuTSaDNQ6dCQA4rou1uxUcFxlBqWlpX0Kb3tnAZRQlSUzFn781Lp8iqAkZmDYqONY93uuYHbMOVuybdWWFI6tiaI15cL1fGQsDwMqIhhQbmJ1YxL1w+SzL53fDEU3oGgGG3JV+Mg1LoMx7NQjAxAy9JZlwTCMbnalJzLEjJ6gLFy2EcvWd3HNnPIYpHom1I1nb4yat6nBbc45BZBd4WNdUwaja2LcSLdmawp1Q0u6lacveXQ7YESgqga85G7kNi5hh8K1skcGIAejT7bu9fP2hGb59b9dCiOakH1VqgbPdTB+4gQYho6c5XMG4OfnlOcfsXZ7Cq1JB7WVUi12pBwOQqaNLIy55JFtgBEFNANTIh/g9bdWQNUN2Ln0V4D0BKsYEMuyMfOBZVz3pi4VzTCZERMmTkTENJC1fdiOjZ+f1bca7G0yMEP0KKDqHPGvfOVPnF3IpTq/AqSnwBo7yDeShn3xGx/glfdaoUcScLMpGPFSeHYOdZPqETUjyHkEiAvbskE5M4pbuN3Uoz1gUR09qCBS9E/35FIG9XWpJht2XTi45tQyzH3gwa9UVk8wmpM+p89JcLlcjtkRS5RzDEJNchR7UF3khLqTUVkShcVLFwgUn7MAVM51bALDwR1n10jPTADVCYFF73Vh8vAELE/FnS82AXoEECrUrs1wGt/h0oDvOl8xpBgUUlckwJxl44e/W8oAUHyQSbazjtf1KFw7i5JjpqB+WBw2LbahP1fAdmQtxbZdpHZvwdiRtYiZcvnDkpWbYVQMhuNYiIgU/FQbxyCtG95GIhZljy3s7Dgi3N4DMerkXfGiHc/HzHnLcOfFx+KXL++Ek0vDziRhxMpkccnOoOSY6Zg0PCqXqBEgVEty5Z5Asj2PF/0QQFTvoPZTT6gcKDquCzudhNK8ijPNqeaN3IxNWQWK2L8CBEATxR+B3aAIm2wCRe+rm4AnX1zKLumAo/tj1552tiEnTvkGrx9xgpVP1FNN7UCUMqFlDTatwGKgZI2eP3O+S66U8l367CCmA75jI7PuZbjJXazfjnhAdnYFbm4ISMCSTCaNLtfETxbvgWpEceP0cjy3fAdarQi+NraKc08OpVlYZQHbPvoA6c7W7mTknFWhYEVtqOl+J0Ixyzi3FTeCRmBK7Tg5RNvWHrmAUBm4LVOIOUKGkIBvmb8G/q73kB1xAVQjxt3pc84tw08Xt6CsvAz1Qw1a/ASHclXEDA9Yv+oNZpLculcNw6MtjR/CGFSHyMBxvAqYjwe1Y279cXJHJiCUUORun55JRVqB63sMyL9/sz/uWnmUbDld+0d4I85GrKwap4+K8KpZAoJVFu09YGPDG4GAqWOeEvAhNhKc1m3rOdo3Bk5EpPYklMRUXgonW7eD5dbeEaiyaHmzz53o+wIiV+IKvL9xGwb3L8Xs16Psiqb//hgS9Vcinkhg6jBT2oLAoBMYZCs+aljGwd35U0bhlJNG9uJHCPxg1q9hDqpH9cg6uYYkbFghXRY0zB9RNmQPrTVnW9E3IKGnRUK/6cUsxwdmy7vwqsfhG2PLpCcWFKPIiyJ1RQb7o9VvcGr+/KmjGRDLpcWcPrKWi4oE5a1U/GDWvagePQWx2hOCxrx8f16wzuEIYkhbJmiACAEJah/FdZB86j1QZTNfSMvKX7IV0bKjMP3YeB4Q8p5CdUWAbGpYyot7zp92LE45aVSohfLWhFTfrXPmITp4AmIDjpd1EK4YFtmcI6Ue0p4Nu1GkDf00hoS25YYXUlKwisJ9XFOGk7qS7mvepSXj7gleWpBq342YYqOmIsZLGmTTNThwjMd0bNmdhllRA80sCf5tSaHvUjay4stv1HltB1XtiiqCvOy5DxsSMob2D7ydxcYWj/W9qamYOtzkYJD//QYHfTL4I3Asqo3QntIoLoEh14SQZ0vNEyGwFPnnVzBT3BGsfaPiFP2o/wUkIE9W9Gd1TQAAAABJRU5ErkJggg==";
```

Comment:
Do we still use this `imgix-loader` after using Cloudinary?

Reply:
No. I left the loader there in case needed, but it can be safely deleted.

Reply:
Do we need to update the `bannerBlurData` here?

Reply:
It has been updated @zainfathoni . I updated it manually to string base64 because this will be rarely updated and always loaded first.

For additional images, now we can programmatically generate the blurred version like so:

```jsx
            <Image
              blurDataURL={getBlurred("v1627049958/some_image.png")}
              height={288}
              layout="responsive"
              loader={cloudinaryLoader}
              placeholder="blur"
              priority={true}
              quality={90}
              src="v1627049958/some_image.png"
              width={640}
            />
```